app/window.py

```diff
@@ -557,15 +557,16 @@ def render(self):
     rowPercentage = 0
     colPercentage = 0
     lineCount = len(tb.lines)
-    if lineCount:
-      rowPercentage = self.host.cursorRow * 100 / lineCount
-      if self.host.cursorRow >= lineCount - 1:
-         rowPercentage = 100
-      charCount = len(tb.lines[self.host.cursorRow])
-      if (self.host.cursorCol < charCount):
-        colPercentage = self.host.cursorCol * 100 / charCount
-      else:
-        colPercentage = 100
+    # Assumes lineCount is always at least 1.
+    if lineCount == 1:
+      rowPercentage = 100
+    else:
+      rowPercentage = (self.host.cursorRow + 1) * 100 / lineCount
```

Comment:
Oh, I think what I was asking about was not doing the +1, so row and column would both be the same equation: number * 100 / count. It means that column 100% is not reachable.

But I'll try this for awhile as-is. Then maybe we can try what I was wondering about.